OrbitGl/PickingManager.h

```diff
@@ -53,6 +53,10 @@ enum class PickingType : uint32_t {
 enum class BatcherId : uint32_t { kTimeGraph, kUi };
 
 struct PickingId {
+  static constexpr const uint32_t kIDBitSize = 28;
```

Comment:
since this is constexpr you can remove static. s/kIDBitSize/kIdBitSize/ - same below

Reply:
No, constexpr members can't be static, at least the VS compiler complains. I can remove the const though, that's kind of redundant.